dashboard/app/helpers/script_levels_helper.rb

```diff
@@ -70,4 +70,51 @@ def tracking_pixel_url(script)
       CDO.code_org_url "/api/hour/begin_#{script.name}.png"
     end
   end
+
+  def hoc_finish_url(script)
+    if script.name == Script::HOC_2013_NAME
+      CDO.code_org_url '/api/hour/finish'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.